pkgs/development/libraries/libdatachannel/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, srcOnly
+, cmake
+, ninja
+, pkg-config
+, libnice
+, openssl
+, plog
+, srtp
+, usrsctp
+}:
+
+let
+  # Use usrsctp version specified at https://github.com/paullouisageneau/libdatachannel/tree/master/deps
+  # Older or newer usrsctp might break libdatachannel, please keep it synced with upstream.
+  customUsrsctp = usrsctp.overrideAttrs (finalAttrs: previousAttrs: {
```

Comment:
A more canonical name we would use would be ``usrsctp'``

Reply:
we can revert this If you find it more then a slight annoyance.

Reply:
I can open another PR if you say this is important.

Reply:
Just keep it in mind for next time or the next change to this file